### ISSUE TITLE
SC1091 is not in the table/documentation

### ISSUE THREAD

**Issue by atombrella** [hadolint/hadolint#1066]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior

SC1091 is not the table in the readme-file (documentation). I don't know if there are more missing rules. It came up as an info alert for using `. /opt/venv/bin/activate` in a Dockerfile to install azure-cli.

Perhaps there are more missing rules in the documentation, as it contains gaps between the rule numbers.

**Comment by DanielChenTO** [comment]
Rules that start with `SC` are documented in ShellCheck repo, the rule you mention is listed here: https://github.com/koalaman/shellcheck/wiki/SC1091

The Documentation here only mentions the commonly used ones.

**Comment by nijel** [comment]
IMHO SC1091 makes no sense at all when shellcheck is invoked from hadolint, that's why this issues pops again and again (https://github.com/hadolint/hadolint/issues/905, https://github.com/hadolint/hadolint/issues/818, https://github.com/hadolint/hadolint/issues/624). Maybe it should be disabled by hadolint? Or is it really useful in some situations?

The proper way to deal with this is to ignore it:

```dockerfile
# hadolint ignore=SC1091
RUN source /opt/venv/bin/activate ...
```

**Comment by tanmay-pnaik** [comment]
Added now: https://github.com/hadolint/hadolint/pull/1088
